gustaf/helpers/init_helper.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any
+
+
+class LibraryCanNotBeLoadedHelper():
+    """
+    Class used to have better import error handling in the case that a package
+    package is not installed. This is necessary due to that some packages are
+    not a dependency of `gustaf`, but some parts require them to function.
+    Examples are `splinepy` and `vedo`.
+    """
+    _message = None
```

Comment:
It actually does not matter since if the variable is overwritten, it is then used as a normal variable of the object. (actually, two variables with the same name would exist and one would be only available as self.__class__._name and the other with higher precedence with self._name

Reply:
Ah good point. Was just confusing and thanks for removing it